ui/shared/status/StatusEmojiPopup.qml

```diff
@@ -120,29 +207,60 @@ Popup {
             height: searchBox.height + emojiHeader.headerMargin
 
             SearchBox {
-               id: searchBox
-               anchors.right: skinToneEmoji.left
-               anchors.rightMargin: emojiHeader.headerMargin
-               anchors.top: parent.top
-               anchors.topMargin: emojiHeader.headerMargin
-               anchors.left: parent.left
-               anchors.leftMargin: emojiHeader.headerMargin
+                id: searchBox
+                anchors.right: skinToneEmoji.left
+                anchors.rightMargin: emojiHeader.headerMargin
+                anchors.top: parent.top
+                anchors.topMargin: emojiHeader.headerMargin
+                anchors.left: parent.left
+                anchors.leftMargin: emojiHeader.headerMargin
             }
 
-            SVGImage {
+            Row {
                 id: skinToneEmoji
-                width: 22
+                property bool expandSkinColorOptions: false
+                width: expandSkinColorOptions ? (22 * skinColorEmojiRepeater.count) : 22
                 height: 22
+                opacity: expandSkinColorOptions ? 1.0 : 0.0
+                Behavior on width { NumberAnimation  { duration: 400 } }
+                Behavior on opacity { NumberAnimation  { duration: 200 } }
+                visible: (opacity > 0.1)
                 anchors.verticalCenter: searchBox.verticalCenter
                 anchors.right: parent.right
                 anchors.rightMargin: emojiHeader.headerMargin
-                source: "../../../../imports/twemoji/72x72/1f590.png"
+                Repeater {
+                    id: skinColorEmojiRepeater
+                    model: ["1f590-1f3fb", "1f590-1f3fc", "1f590-1f3fd", "1f590-1f3fe", "1f590-1f3ff", "1f590"]
```

Comment:
This could be simplified with something like this:
```suggestion
                    model: skinColors.map(function (color) {return "1f590-" + color;}).push("1f590")
```

Reply:
This doesn't work. Probably because the last one is just "1f590" with no color? It could also change to something like:
`model: 6	`
and then: 
`source: "../../imports/twemoji/72x72/1f590" + (index < 5 ? "-" + skincolor[index] : "") + ".png"`
for example, but I preferred to keep things simple as it's a really simple case here.